react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+let counter = 0;
+export default class HotButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.state = { isClicked: counter };
```

Comment:
Here when you set your initial state you can start it at `0`. Also use a property name that makes since. If state is being used as a counter, then call the property counter.
```JSX
this.state = { counter: 0 };
```